src/Integration/Helpers/SonarLintOutputLogger.cs

```diff
@@ -41,21 +41,32 @@ public class SonarLintOutputLogger : ILogger
 
         public void WriteLine(string message)
         {
-            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, message);
+            var prefixedMessage = AddPrefixIfVerboseLogging(message);
+            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, prefixedMessage);
         }
 
         public void WriteLine(string messageFormat, params object[] args)
         {
-            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, messageFormat, args);
+            var prefixedMessageFormat = AddPrefixIfVerboseLogging(messageFormat);
+            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, prefixedMessageFormat, args);
         }
 
         public void LogVerbose(string messageFormat, params object[] args)
         {
             if (sonarLintSettings.DaemonLogLevel == DaemonLogLevel.Verbose)
```

Comment:
The check against the log level is being done twice - once here, and once when the method calls through to `WriteLine`. However, refactoring it would have been uglier, and it isn't performance critical.